### DISCUSSION TITLE
Default map_style pydeck

### DISCUSSION THREAD

**Discussion by AdrianKriger** [visgl/deck.gl#7150] [Q&A] [answered]
Hi.

Can you please tell me what / where the pydeck default map_style (light, road) is?

I want to direct a request to the provider ---it does not render [leisure=sports_centre](https://wiki.openstreetmap.org/wiki/Tag:leisure=sports%20centre?uselang=en).

**Comment by kylebarron** [comment, marked as answer]
The default map style comes from Carto:
https://github.com/visgl/deck.gl/blob/bb1d8accfb68b1866402569437241705b8ee45ea/bindings/pydeck/pydeck/bindings/deck.py#L27

**Reply by AdrianKriger** [reply]
Thank you.
